doc/nrf/working_with_nrf/nrf70/developing/regulatory_support.rst

```diff
@@ -0,0 +1,358 @@
+     - Yes
+   * - 104
+     - 5510
+     - 5530
+     - 5520
+     - U-NII-2C
+     - Yes
+   * - 108
+     - 5530
+     - 5550
+     - 5540
+     - U-NII-2C
+     - Yes
+   * - 112
+     - 5550
+     - 5570
+     - 5560
+     - U-NII-2C
+     - Yes
+   * - 116
+     - 5570
+     - 5590
+     - 5580
+     - U-NII-2C
+     - Yes
+   * - 120
+     - 5590
+     - 5610
+     - 5600
+     - U-NII-2C
+     - Yes
+   * - 124
+     - 5610
+     - 5630
+     - 5620
+     - U-NII-2C
+     - Yes
+   * - 128
+     - 5630
+     - 5650
+     - 5640
+     - U-NII-2C
+     - Yes
+   * - 132
+     - 5650
+     - 5670
+     - 5660
+     - U-NII-2C
+     - Yes
+   * - 136
+     - 5670
+     - 5690
+     - 5680
+     - U-NII-2C
+     - Yes
+   * - 140
+     - 5690
+     - 5710
+     - 5700
+     - U-NII-2C
+     - Yes
+   * - 144
+     - 5710
+     - 5730
+     - 5720
+     - U-NII-2C
+     - Yes
+   * - 149
+     - 5735
+     - 5755
+     - 5745
+     - U-NII-3
+     -
+   * - 153
+     - 5755
+     - 5775
+     - 5765
+     - U-NII-3
+     -
+   * - 157
+     - 5775
+     - 5795
+     - 5785
+     - U-NII-3
+     -
+   * - 161
+     - 5795
+     - 5815
+     - 5805
+     - U-NII-3
+     -
+   * - 165
+     - 5815
+     - 5835
+     - 5825
+     - U-NII-3
+     -
+   * - 169
+     - 5835
+     - 5855
+     - 5845
+     - U-NII-4
+     -
+   * - 173
+     - 5855
+     - 5875
+     - 5865
+     - U-NII-4
+     -
+   * - 177
+     - 5875
+     - 5895
+     - 5885
+     - U-NII-4
+     -
+
+Conformance to regulatory restrictions
+**************************************
+
+Once a regulatory domain is configured, the following restrictions are imposed on the operation of the nRF70 Series devices:
+
+1. Channels: The device will operate only on the channels allowed in that country.
+#. Scan mode: The device will use Active scans where allowed and resort to Passive scans on channels where Active scans are not allowed.
+#. DFS: DFS is enabled on the channel.
+   The device will operate in the following two states:
+
+   * Scan state: The channel operation must be in the Passive mode.
+   * Connected state: The device will switch to other directed channel by an access point in case of radar detection.
+#. Transmit power: The maximum transmit power on each channel is restricted to a value allowed for that channel in that country.
+
+In the absence of a user configured country setting, the nRF70 Series device configures to the world regulatory domain by default.
+The AP beacon message may have the regulatory hints such as country code information, to be considered in the region configuration settings.
+
+
+Configuration options
+*********************
+
+You can configure the regulatory domain through one of the following options:
+
+Build time
+==========
+
+Use the :kconfig:option:`CONFIG_NRF700X_REG_DOMAIN` Kconfig option to set the regulatory region.
+The regulatory region will take a ISO/IEC alpha2 country code for the country in which the device is expected to operate.
```

Comment:
@rajb9: Can you explain this?

Reply:
Kconfig is the build time option. It is not equivalent to the force=true. If user opt for this method, the Beacon's Country code (if present) will be given precedence over the Kconfig option. Otherwise, the chosen Kconfig option is configured.

Reply:
So describe this behaviour explicitly. I think you are saying it is equivalent to force = false described later.

Reply:
Add the following line under **Build time** option:
The beacon's regulatory region (if present) will be given high precedence over the Kconfig option.